docs/inspec_and_friends.md

```diff
@@ -40,6 +40,14 @@ control "sshd-11" do
 end
 ```
 
+That said, InSpec is not RSpec. InSpec does not support arbitrary RSpec features. Some key differences:
+
+ * RSpec `describe` blocks are intended to be nested. In InSpec, `describe` 
+ blocks should not be nested; instead use `control` blocks to describe a 
+ higher-level grouping of tests.
+ * RSpec is a tool designed for software engineers. It thus supports a very large range of matchers, to enable testing of software engineering constructs such as exceptions, Object Oriented Programming relationships, and so on. In contrast, InSpec is aimed at compliance practioners and infrastructure testers, so our focus is providing a few, well-supported, easy-to-use universal matchers, such as `cmp`.
+ * RSpec is a separate project from InSpec. Its release schedule and feature set are beyond the control of the InSpec team. While you are free to use RSpec-specific features in your InSpec profiles, the InSpec team can only support those features described at [docs.inspec.io](https://docs.inspec.io).
```

Comment:
> InSpec is built on top of RSpec and uses it as the underlying foundation to execute tests. It uses the key strengths of RSpec, easily execute tests and a DSL to write tests, but extends the functionality for use as compliance audits. 

For clarity and consistency, I would update DSL to state Domain Specific Language (DSL). 

This is not a part you edited. But if you agree I think it would help the overall document.